extensions/remote-udp/README.md

```diff
@@ -0,0 +1,65 @@
+# Remote UDP Extension
+
+The openEO API is a specification for interoperable cloud-based processing of large Earth observation datasets.
+
+This extension enables user to load user-defined processes that are hosted on other hosts through the process namespace into process graphs.
```

Comment:
"hosted on other hosts" sounds quite generic. And it's also not obvious what "other" means: other than what?

I'd propose to add a more concrete example

> (for example: external file sharing or source hosting services)

Reply:
```suggestion
This extension enables user to load user-defined processes that are hosted external to the openEO API (e.g., GitHub or cloud storage) through the process namespace into process graphs.
```